services/core-web/src/components/homepage/HomeMineActivity.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { FC, useState, useEffect } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Link, useHistory } from "react-router-dom";
+import { Row, Col, Typography, Divider, Skeleton } from "antd";
+import { EnvironmentOutlined, BookOutlined } from "@ant-design/icons";
+import { fetchSubscribedMinesByUser } from "@common/actionCreators/mineActionCreator";
+import { fetchGlobalMineAlerts } from "@/actionCreators/mineAlertActionCreator";
+import { getSubscribedMines, getSubscribedMinesLoaded } from "@common/selectors/mineSelectors";
+import { getGlobalMineAlerts } from "@/selectors/mineAlertSelectors";
+import { formatDateTime } from "@common/utils/helpers";
+import { IMine, IMineAlert } from "@mds/common";
+import * as routes from "@/constants/routes";
+import { RootState } from "@/App";
+
+interface HomeMineActivityProps {
+  subscribedMines: IMine[];
+  subscribedMinesLoaded: boolean;
+  mines: IMine[];
+  alerts: IMineAlert[];
+  alertsLoaded: boolean;
+  fetchSubscribedMinesByUser: any;
```

Comment:
```suggestion
  fetchSubscribedMinesByUser: ActionCreator<typeof fetchSubscribedMinesByUser>;
```
For any functions retrieved from action creators, we can use the ActionCreator interface like this.  That way it will properly interpret the properties and such of the action creator.

Reply:
nice, fixed this in 2 files